### PR TITLE
Include Split packages

### PR DESCRIPTION
**Pull request by javiercn**

### CODE REVIEW

**Comment by javiercn** [comment]
@pranavkm

File: build/project.json [hunk 1]
```diff
@@ -6,6 +6,7 @@
       "Sake": "0.2.2",
       "xunit.runner.console": "2.1.0",
       "xunit.core": "2.1.0",
-      "NuGetPackageVerifier": "1.0.1-*"
+      "NuGetPackageVerifier": "1.0.1-*",
+      "SplitPackages": "1.0.1-*"
```
Comment (pranavkm) [inline build/project.json:10]: Did you decide against adding it to Coherence? This would affect everybody's builds marginally.

**Comment by javiercn** [comment]
@victorhurdugaci Are you ok? We need this for Coherence

**Comment by pranavkm** [comment]
:shipit: if @victorhurdugaci is ok.

**Comment by victorhurdugaci** [comment]
Is it needed for every repo or just coherence?

**Comment by javiercn** [comment]
@victorhurdugaci Coherence, but it's the same thing we do with NugetPackageVerifier and it's darn cheap (parallel + cached) so no big impact.

**Pull request closed without merging**